### DISCUSSION TITLE
将一个已经编译好的dll作为target引入？

### DISCUSSION THREAD

**Discussion by VirFunc** [xmake-io/xmake#2333] [Q&A] [answered]
我有一个dll和它对应的 header 和 lib，怎么把它作为一个 target 引入作为其他 target 的依赖。

**Comment by waruqi** [comment]
add_deps

**Reply by VirFunc** [reply]
> add_deps

这个 dll 是三方库的，不是由 xmake 构建的

**Comment by waruqi** [comment, marked as answer]
add_links add_linkdirs

**Reply by VirFunc** [reply]
直接 link 这个 dll 我知道要这样做，就像我标题里写的，但是能不能把它单独作为一个 target 呢 (或者是一个本地的 package?)。
在CMake 里是可以通过这个 [imported libraries](https://cmake.org/cmake/help/latest/command/add_library.html#imported-libraries) 来实现的。

**Reply by waruqi** [reply]
不能 还是刚说的 走 add_links

**Reply by VirFunc** [reply]
好吧，谢谢解答
